.circleci/config.yml

```diff
@@ -71,7 +71,7 @@ jobs:
             TERM: dumb
       steps:
         - checkout
-        - run: yarn install
+        - run: yarn --frozen-lockfile
```

Comment:
probably we can get rid of both checkout and yarn --forzen-lockfile

Reply:
Removed install, checkout needs to stay as we discussed.